### ISSUE TITLE
Freezing glove word embeddings for few epochs

### ISSUE THREAD

**Issue by 404akhan** [bentrevett/pytorch-sentiment-analysis#28]
I saw that in some papers, people freeze embedding weights, before further layers converge to something meaningful. After that they unfreeze them and train the whole network together. What do you think about adding this feature, say as an appendix?

**Comment by bentrevett** [comment]
Great idea. I think I'll do something like freezing the embedding weights until the validation loss stops decreasing and then unfreeze the embedding weights. Sound good?

**Comment by 404akhan** [comment]
Sounds reasonable, yeah try it. Although most papers i've seen did different things. But this also looks good, at least seeing basic functionality of freeze-unfreeze under certain condition will be already good for people using tutorial.

**Comment by bentrevett** [comment]
OK, I've implemented a [notebook appendix C](https://github.com/bentrevett/pytorch-sentiment-analysis/blob/master/C%20-%20Loading%2C%20Saving%20and%20Freezing%20Embeddings.ipynb) which covers how to freeze/unfreeze embeddings (and a few other things).

In the implementation it freezes the embeddings for the first 5 epoches and then unfreezes them for the next/last 5. I also added a code snippet on how to unfreeze when the validation loss stops decreasing.

Let me know what you think.
